ui/src/app/cohort-search/search-group-list/search-group-list.component.ts

```diff
@@ -23,7 +23,7 @@ export class SearchGroupListComponent {
   }
 
   get title() {
-    const verb = this.role.charAt(0).toUpperCase() + this.role.slice(1, -1) + 'd';
-    return `${verb} Participants`;
+    const verb = this.role.charAt(0).toUpperCase() + this.role.slice(1, -1);
+    return `${verb} Participants Where`;
```

Comment:
You'll need to update the test for this (or remove it - arguably, it just tests copy, and its an open question as to whether or not we should test copy.  What do you think?)

Reply:
Removed.